models/platform-limits/src/business-units/types.ts

```diff
@@ -0,0 +1,19 @@
+import type { TBuilder } from '@commercetools-test-data/core';
+import type { TLimit } from '../limit';
+
+export type TBusinessUnitLimitsProjection = {
+  maxDivisions: TLimit;
+  maxDepthLimit: TLimit;
+  maxAssociates: TLimit;
+  maxAssociateRoles: TLimit;
+};
```

Comment:
I couldn't find the `type` in any of the repo. I wrote them down based on the schema from graphql explorer
<img width="407" alt="Screenshot 2024-08-28 at 08 44 19" src="https://github.com/user-attachments/assets/d695bd5b-660a-4335-8275-a2c4450480bb">

Reply:
Fair enough 👍